lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -276,6 +276,10 @@ export abstract class ClientApplication {
                     this.nativeExtensionProvider = undefined; // If extension gets uninstalled during session prevent future requests from continuing to attempt
                     const redirectClient = this.createRedirectClient(request.correlationId);
                     return redirectClient.acquireToken(request);
+                } else if (e instanceof InteractionRequiredAuthError) {
+                    this.logger.verbose("acquireTokenRedirect - Resolving interaction required error thrown by native broker by falling back to web flow");
```

Comment:
I think it would be worth reporting this in our telemetry.  A boolean to indicate this has occurred would be enough.

Reply:
I believe this will be addressed in the additional Telemetry work I have on my radar for JS-WAM bridge